akka-docs/src/test/scala/docs/persistence/testkit/TestKitExamples.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.persistence.testkit
+
+import akka.actor.typed.ActorSystem
+import akka.persistence.testkit._
+import akka.persistence.testkit.scaladsl.PersistenceTestKit
+import akka.persistence.typed.scaladsl.{ Effect, EventSourcedBehavior }
+import com.typesafe.config.ConfigFactory
+import org.scalatest.WordSpecLike
+
+class TestKitExamples {
+
+  //#testkit-typed-usecase
+  class TypedSampleSpec extends WordSpecLike {
+
+    val system: ActorSystem[Cmd] = ActorSystem(
```

Comment:
better to use ScalaTestWithActorTestKit
https://doc.akka.io/docs/akka/current/typed/testing-async.html?language=scala#test-framework-integration